src/PointwiseFunctions/AnalyticData/Xcts/Binary.hpp

```diff
@@ -242,6 +249,16 @@ struct BinaryVariables
  * The falloff can be disabled by passing `std::nullopt` to the constructor, or
  * `None` in the input file.
  *
+ * \par Matter sources
```

Comment:
Could you document the `windows` stuff somewhere? I don't see where the `windows` variable is documented, nor the `ApplyWindow` bool. I'm just commenting here, but I'm not sure where the right place to ducement the window thing is...

Reply:
The paragraph above this line documents the Gaussian falloffs, are you missing something there?

Reply:
I guess something is missing connecting that paragraph to the `window` name in the code? At least that's what I'm looking for. I was hoping from searching for `window` in the hpp that something would say something like `Window is the falloff described in the documentation`

Reply:
👍 Added a fixup

Reply:
Perfect, thanks! Please squash!